proc/dsrg.py

```diff
@@ -78,6 +109,7 @@ def __init__(self, active_space_solver, state_weights_map, mo_space_info, ints,
         self.Heff_implemented = False
         self.converged = False
         self.energies = []  # energies along the relaxation steps
+        self.energies_environment = {}  # energies pushed to Psi4 environment
```

Comment:
Can you change the comment to "Psi4 environment **globals**"? That extra word makes the comment much clearer to me.